package.json

```diff
@@ -49,7 +49,7 @@
     "test": "npm run test:main && npm run test:fp",
     "test:fp": "node test/test-fp",
     "test:main": "node test/test",
-    "validate": "npm run style & npm run test"
+    "validate": "npm run style & npm run test & wait"
   },
```

Comment:
Alternatively, you might try npm.im/npm-run-all. I've had great success with that in my projects.